File: hackerrank/problem solving/algorithms/1 - warmup/1 - solve me first/with morfo library/solve_me_first.java
// https://www.hackerrank.com/challenges/solve-me-first/problem?isFullScreen=true
// Java 25

import morfo.io.Reader;

void main() {
    int n1 = Reader.read(Integer::parseInt);
    int n2 = Reader.read(Integer::parseInt);
    IO.println(solveMeFirst(n1, n2));
}

// T: O(1)
// S: O(1) extra space
int solveMeFirst(int n1, int n2) {
    return n1 + n2;
}

File: hackerrank/problem solving/algorithms/1 - warmup/1 - solve me first/with morfo library/solve_me_first.cpp
// https://www.hackerrank.com/challenges/solve-me-first/problem?isFullScreen=true
// C++23

import morfo;
import std;
using namespace std;

int solve_me_first(int n1, int n2);

int main()
{
    auto [n1, n2] = morfo::read<int, int>();
    cout << solve_me_first(n1, n2);
    return 0;
}

// T: O(1)
// S: O(1) extra space
int solve_me_first(int n1, int n2)
{
    return n1 + n2;
}
